packages/passport/sdk/src/react/PassportContext.tsx

```diff
@@ -0,0 +1,279 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
There's quite a bit of functionality within this module - can we please add some tests?

Reply:
I'll need to address all the feedback first.